tools/wpt/browser.py

```diff
@@ -379,14 +379,14 @@ def _latest_geckodriver_version(self):
         tags = call("git", "ls-remote", "--tags", "--refs",
                     "https://github.com/mozilla/geckodriver.git")
         release_re = re.compile(r".*refs/tags/v(\d+)\.(\d+)\.(\d+)")
-        latest_release = 0
+        latest_release = []
```

Comment:
It probably doesn't matter much, but I think I'd prefer `latest_release = (0,)` here and later on `version = tuple(int(iitem) for item in m.groups())` and `assert latest_release != (0,)`

Reply:
Can you explain why? I'm not very familiar with python, so curious as to why tuples are better for this case.

Reply:
I think mainly what I like is putting an explicit zero in the default value, so that you don't need to worry about how empty sequences compare and can just remember that sequence comparison is performed element-wise. Arguably `(0,0,0)` is even better in this case as the default since we know geckodriver follows Rust conventions which are semver-based so we expect to have (major, minor, patch) in the version number (and indeed the regex above produces exactly 3 fields of output).

 But on tuples vs lists, conceptually tuples are supposed to be a compact representation of data containing several fields (usually anonymous, but `NamedTuple` changes this), whereas a list is supposed to be an ordered, (possibly) hetrogenous, collection of items. Conceptually a version identifier seems more like the former than the latter. That said, I accept this may be angels-on-the-heads-of-pins stuff and it doesn't really matter in this case.